packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -393,9 +428,24 @@ class DebuggerScreen extends Screen {
       }
     });
 
-    CoreElement scriptCountDiv;
+    final CoreElement textfield =
+        CoreElement('input', classes: 'form-control input-sm')
+          ..setAttribute('type', 'text')
+          ..setAttribute('placeholder', 'script_name')
+          ..element.style.width = '75%'
```

Comment:
Would like a better mechanism for centering the InputElement May flex or something better?